suite-common/icons/generateIcons.js

```diff
@@ -1,11 +1,14 @@
 /* eslint-disable import/no-extraneous-dependencies */
 
 import chalk from 'chalk';
-
-const fs = require('fs');
-const path = require('path');
-const prettier = require('prettier');
-const { optimize } = require('svgo');
+import fs from 'fs';
```

Comment:
no need this change

Reply:
This issue was addressed during the rebase.